apps/api/src/deployment/controllers/deployment/deployment.controller.ts

```diff
@@ -86,4 +88,31 @@ export class DeploymentController {
 
     return { data: result };
   }
+
+  @Protected([{ action: "sign", subject: "UserWallet" }])
+  async list({ skip, limit }: { skip?: number; limit?: number }): Promise<z.infer<typeof ListDeploymentsResponseSchema>> {
+    const { currentUser, ability } = this.authService;
+
+    const userWallet = await this.userWalletRepository.accessibleBy(ability, "sign").findOneByUserId(currentUser.id);
+    assert(userWallet, 404, "UserWallet Not Found");
+    assert(
+      (skip === undefined && limit === undefined) || (skip !== undefined && limit !== undefined),
+      400,
+      "Skip and limit must be provided together or not at all"
```

Comment:
why can't limit and skip be used independently?

Reply:
I guess we can make independent, I was thinking in terms of pagination it doesn't make sense